components/ProductListItem/index.js

```diff
@@ -1,14 +1,28 @@
 import styled from "styled-components";
+import Link from "next/link";
 
-const StyledProductListItem = styled.li`
+const StyledProductListItem = styled.div`
   width: 100%;
   background-color: var(--secondaryBackgroundColor);
   padding: 10px;
   margin-block: 7px;
   border-radius: 5px;
-  box-shadow: 0px 1px 3px var(--primaryFontColor);
+  box-shadow: 0px 1px 3px var(--primaryDarkColor);
+  &:hover {
+    cursor: pointer;
+  }
+`;
+const StyledLink = styled(Link)`
+  text-decoration: none;
+  color: var(--primaryDarkColor);
 `;
 
 export default function ProductListItem({ product }) {
-  return <StyledProductListItem>{product.name}</StyledProductListItem>;
+  return (
+    <li>
+      <StyledLink href={`/products/${product._id}`}>
```

Comment:
Apply the styling directly to the `<li>` element while maintaining the same styling and ensures semantic clarity. ✨

Reply:
Hi Jessica, thanks for your great suggestion. We tried to apply directly the styling to the <li> but then the padding was not clickable. So we decided to put the complete Styling into the Link element.